src/Codeception/Module/Redis.php

```diff
@@ -620,12 +623,16 @@ private function checkKeyContains(string $key, $item, $itemValue = null): bool
      * @param mixed  $value Optional. If specified, also checks the key has this
      * value. Booleans will be converted to 1 and 0 (even inside arrays)
      */
-    private function checkKeyExists(string $key, $value = null): bool
+    private function checkKeyExists(string $key, $value, bool $throwException = false): bool
```

Comment:
I thought you didn't like parameter flags

Reply:
I think it is ok for internal API... But not for public

Reply:
I came up with a way to avoid flag parameter :)

Reply:
Sweet! thanks